### DISCUSSION TITLE
Argus Tower and Thank you

### DISCUSSION THREAD

**Discussion by surfing-in-the-sun** [sdcTools/UserSupport#210] [User Group]
@AncoHundepool  was so kind to provide the Link to see the  **Argus tower** monitoring the coast very near to The Hague:

https://en.wikipedia.org/wiki/Argus_Coastal_Monitoring

**Thank you all for a great first User Meeting today. We will provide the presentation material and report in the next days.**

Best regards, Paula

**Comment by ppdewolf** [comment]
So, let's try to convert our User Group into a similar "global network of Argus monitoring stations" 😉
